src/webportal/src/app/user/fabric/batchRegister/TopBar.jsx

```diff
@@ -37,7 +37,7 @@ function TopBar() {
     iconProps: {
       iconName: 'OpenFolderHorizontal',
     },
-    buttonStyles: {root: buttonPrimaryStyles},
+    buttonStyles: {root: [buttonMarginStyle, {marginLeft: `-${spacing.s1}`}]},
```

Comment:
I think we'd better use CommandBar's styles -> root to set the horizontal padding to spacing.s1 (8px) and use margin right to set the spacing between buttons.
Another option is to use a basic Stack instead of CommandBar.

Reply:
Good suggestion. Fixed.